src/Domain/Entities/ProductDetail.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities
+{
+    public class ProductDetail
+    {
+        public Guid Id { get; set; }
+        public Guid ProductId { get; set; }
+        public decimal ProductSellingPrice { get; set; }
```

Comment:
Na właściwości i pola patrzymy kontekstowo. Jeśli zobaczę pole decimal z `price` w nazwie w klasie `ProductDetail`, a taką właściwość w klasie `Product` to raczej będę wiedział, że to cena tego produktu :) Krótkie nazwy w klasach domenowych robią robotę, lepiej użyć krótkiej nazwy, niż długiej jeśli są tak samo dobre